catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -536,27 +525,53 @@ public void run() {
 
         public CallableSourceResponse(Source source, QueryRequest request) {
             this.source = source;
-            this.request = request;
+            this.request = new QueryRequestImpl(request.getQuery(), new HashMap<>(request.getProperties()));
         }
 
         @Override
         public SourceResponse call() throws Exception {
-            final SourceResponse sourceResponse =
-                    source.query(new QueryRequestImpl(request.getQuery(), request.getProperties()));
+            SourceResponse sourceResponse = null;
+            try {
+                sourceResponse = source.query(new QueryRequestImpl(request.getQuery(),
+                        request.getProperties()));
+            } catch (UnsupportedQueryException e) {
+                LOGGER.info("Error querying source {}", source.getId(), e);
+            }
+
+            QueryResponse queryResponse = new QueryResponseImpl(request,
```

Comment:
Or maybe this is something that should be done at the catalog level instead of the source level?  Or maybe we just don't want that behavior period. do @lessarderic @pklinef  have thoughts on this? I think we discussed something similar before but dont remember what the outcome was.